tests/contrib/minikube/test_kubernetes_pod_operator.py

```diff
@@ -326,6 +330,69 @@ def test_xcom_push(self):
         )
         self.assertEqual(k.execute(None), json.loads(return_value))
 
+    def test_envs_from_configmaps(self):
+        # GIVEN
+        return_value = 123
+        configmap_name = 'test-configmap'
+        metadata = V1ObjectMeta(
+            name=configmap_name,
+            namespace='default'
+        )
+        configmap_object = V1ConfigMap(
+            data={
+                "TEST_CONFIGMAP_VALUE": str(return_value)
+            },
+            metadata=metadata
+        )
+        client = get_kube_client(in_cluster=False)
+        client.create_namespaced_config_map(namespace='default', body=configmap_object)
```

Comment:
This leaves the ConfigMap object hanging around in kubernetes, which means if you try to run the same test twice in the cluster it will fail the second time around.

Please use `try`/`finally` to remove the config map at the end. (Similarly for the Secret).

Alternatively I would accept not actually running the pod (as this started to get in to the realm of testing behaviour of kubernetes itself) but instead mocking and checking that we pass the expected structure to kube clients etc.

Reply:
(My main concern with actually running the KubePodOperator here is that it takes extra time compared to just mocking, and we don't need to check that kube does what it says with valueFrom etc - kube has it's own test to check that)

Reply:
Example  where we are mocking  already in tests https://github.com/apache/airflow/blob/34ad533cac3a13d370de9ddced72312268e33511/tests/contrib/minikube/test_kubernetes_pod_operator.py#L67-L89